pex/bin/pex.py

```diff
@@ -503,8 +514,12 @@ def main():
     args, cmdline = args, []
 
   options, reqs = parser.parse_args(args=args)
+  if options.pex_root:
+    ENV.set('PEX_ROOT', options.pex_root)
+  options.cache_dir = make_relative_to_root(options.cache_dir)
+  options.interpreter_cache_dir = make_relative_to_root(options.interpreter_cache_dir)
 
-  with ENV.patch(PEX_VERBOSE=str(options.verbosity)):
+  with ENV.patch(PEX_VERBOSE=str(options.verbosity), PEX_ROOT=options.pex_root):
```

Comment:
what happens if `options.pex_root` is `None` here? does that clobber the default `PEX_ROOT`?